airflow/www/static/js/calendar.js

```diff
@@ -63,9 +63,6 @@ const dateFormat = 'YYYY-MM-DD';
 document.addEventListener('DOMContentLoaded', () => {
   $('span.status_square').tooltip({ html: true });
 
-  // JSON.parse is faster for large payloads than an object literal
-  const rootData = JSON.parse(calendarData);
```

Comment:
We want to keep this `JSON.parse` -- it's a _huge_ performance improvement for large DAGs.

Reply:
See #7492

Reply:
How so? It is already a json object at this point... Or should we just not use `htmlsafe_json_dumps()`?

Reply:
Explanation here: https://v8.dev/blog/cost-of-javascript-2019#json

Reply:
And it makes sense

Reply:
It's detailed in that PR. The point is that an object literal slows down DOMReady, so delaying it to after the event via `JSON.parse` makes it more performant. So yes, `const calendarData = {{ data|tojson }};` makes calendarData a string that. That's the point.)

(The variable is badly named.)

Reply:
(Thanks GitHub for not showing me Jarek's comments.)

Reply:
Ok I see. My bad. I guess I forgot that runtime is different than when jinja is parsing.